pkg/trtl/config/config.go

```diff
@@ -180,3 +169,48 @@ func (c MTLSConfig) ParseTLSConfig() (_ *tls.Config, err error) {
 		ClientCAs:  certPool,
 	}, nil
 }
+
+func (c MTLSConfig) GetCertPool() (_ *x509.CertPool, err error) {
+	if c.pool == nil {
+		if err = c.load(); err != nil {
+			return nil, err
+		}
+	}
+	return c.pool, nil
+}
+
+func (c MTLSConfig) GetCert() (_ tls.Certificate, err error) {
+	if len(c.cert.Certificate) == 0 {
+		if err = c.load(); err != nil {
+			return c.cert, err
+		}
+	}
+	return c.cert, nil
+}
+
+func (c MTLSConfig) load() (err error) {
```

Comment:
grrr …. debugging for an hour and it turns out this is a pointer error … the linter even saw it below … 🤦🏽